md4c/md4c.c

```diff
@@ -139,6 +139,7 @@ struct MD_CTX_tag {
 #define UNDERSCORE_OPENERS                      ctx->mark_chains[8]
 #define TILDE_OPENERS                           ctx->mark_chains[9]
 #define BRACKET_OPENERS                         ctx->mark_chains[10]
+#define DOLLAR_OPENERS                          ctx->mark_chains[11]
```

Comment:
Update `OPENERS_CHAIN_LAST` to 11. This is needed to some code which needs to operate on all the opener chains in a generic way.